Backend/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is this file strictly needed even if it's empty?

Reply:
I think it is used for writing app specific configuration, e.g port in which the server is going to run... I think I can remove it.